README.md

```diff
@@ -214,45 +209,55 @@ Calls back with an object of the form:
 }
 ```
 
-### `crypto.keys.marshalPublicKey(key[, type], callback)`
+### `crypto.keys.marshalPublicKey(key, [type])`
 
 - `key: keys.rsa.RsaPublicKey | keys.ed25519.Ed25519PublicKey | require('libp2p-crypto-secp256k1').Secp256k1PublicKey`
 - `type: String`, see [Supported Key Types](#supported-key-types) above.
```

Comment:
What does type default to if not specified?

Reply:
`rsa`, will update the docs

Reply:
Done!